app/src/sharedTest/java/org/oppia/android/app/devoptions/ViewEventLogsFragmentTest.kt

```diff
@@ -170,116 +169,6 @@ class ViewEventLogsFragmentTest {
     }
   }
 
-  @Test
-  fun testViewEventLogsFragment_checkRecentLogsAreDisplayedFirst() {
```

Comment:
Should this & the next test instead be verifying a different view is first?

Reply:
Removed these two by mistake, restored now -- thanks!